sky/sdk/lib/widgets/variable_height_scrollable.dart

```diff
@@ -14,18 +12,32 @@ class VariableHeightScrollable extends Scrollable {
   VariableHeightScrollable({
     String key,
     this.builder,
-    this.token
+    this.token,
```

Comment:
I don't really understand how the token interacts with the layoutState. Can you document when you're supposed to use one, or the other, or whatnot? What are you supposed to do when the list changes? When do you set a new token? What does updating the layoutState and setting a new token do?